src/platform/monitoring/DowntimeNotification/README.md

```diff
@@ -0,0 +1,19 @@
+# Downtime Notifications
+Downtime Notications provide a method of handling service outages in front-end applications. For example, if your application _My App_ leveraged a service called _Fancy Service_, and _Fancy Service_ was known to be offline for maintenance every now and then, it would be desirable to show a message to the user explaining that _My App_ is unavailable at this time so that users aren't given the impression that they are running into unexpected errors and so that _Fancy Service_ isn't needlessly hit. This should also be easy for engineers to do, without requiring any deployments to publish/unpublish the messaging.
+
+This was implemented in Downtime Notifications via a React component intended to wrap an application's entry point. After a user lands on the route for that React app, the Downtime Notification React component will fetch the maintenance windows from the VA API, which are registered in [Pager Duty](https://dsva.pagerduty.com/maintenance_windows). If there are any active maintenance windows for the application's registered dependencies, then instead of rendering the application as usual, the application will render a message explaining to the user that the application is undergoing maintenance and the timeframe for which it will last.
+
+## Implementation
+In general, the process for configuring this in a front-end application looks like:
+
+1. Your team identifies your application's dependencies
```

Comment:
- can use implied 2nd person here e.g. 

1. Identify the application service dependencies 

---

- break these into discrete steps  e.g.

2. Add the services to the externalServices object? list? 

_This value must match the corresponding value in the devops config_

Reply:
also recommend inlining specific examples if they can fit without significantly breaking up the content

Reply:
I think we're still working through the style guide so take these suggestions with a grain of salt. We can also help you edit this if you don't have time to make these changes. 

This is a good example of what we're angling for: https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started#node-setup